cheetah/accelerator/rbend.py

```diff
@@ -51,6 +55,7 @@ def __init__(
             fringe_integral=fringe_integral,
             fringe_integral_exit=fringe_integral_exit,
             gap=gap,
+            gap_exit=gap_exit,
             name=name,
             device=device,
             dtype=dtype,
```

Comment:
@cr-xu we are struggling with this PR a bit and I think you might have useful input.

In `RBend` there might be an issue with `e1` and `e2`. If I instantiate an `RBend` and then access `my_rbend.e1`, the `e1` I get is different from the one I set in the instantiation, by being in `Dipole` space rather than `RBend` space. That's an issue, right?

@Hespe and I were brainstorming solutions and one idea was to define two different `e1` and `e2` ... something like `dipole_e1` and `rbend_e1`, where `rbend_e1` would just be a computed property of the `RBend` class that computes based on `dipole_e1`.

What are your thoughts on this? Are you aware of any conventions already present in other codes or in general notation?